clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -521,6 +522,20 @@ mlir::Value CIRAttrToValue::visitCirAttr(cir::GlobalViewAttr globalAttr) {
   llvm_unreachable("Expecting pointer or integer type for GlobalViewAttr");
 }
 
+// TypeInfoAttr visitor.
+mlir::Value CIRAttrToValue::visitCirAttr(cir::TypeInfoAttr typeinfoArr) {
```

Comment:
rename `typeinfoArr` to `typeInfo`

Reply:
Should we rename it to `typeInfo` or `typeInfoAttr` similar to other parameters in visitors?

Reply:
Yeah `typeInfoAttr` is probably even better :)